ethereum/ethstats/src/main/java/org/hyperledger/besu/ethstats/util/NetstatsUrl.java

```diff
@@ -48,8 +53,9 @@ static NetstatsUrl fromParams(final String url, final String contact) {
             .nodeName(netStatsUrl.group(1))
             .secret(netStatsUrl.group(2))
             .host(netStatsUrl.group(3))
-            .port(Integer.parseInt(Optional.ofNullable(netStatsUrl.group(5)).orElse("3000")))
+            .port(Integer.parseInt(Optional.ofNullable(netStatsUrl.group(5)).orElse("-1")))
```

Comment:
What's the impact of switching from 3000 to -1 here?

Reply:
The usage of getPort is in EthStatsService class. There we can derive the 443/80 ports if this port is set to -1. The other option might be to use Option<Integer> to represent the port instead of -1.

Reply:
I see the breaking change mentioned in the changelog now 👍